CHANGELOG.md

```diff
@@ -1,15 +1,40 @@
 ## master / unreleased
 
-* [CHANGE] Improve filter flag names.
 * [CHANGE]
 * [FEATURE]
-* [ENHANCEMENT] Include TCP OutRsts in netstat metrics
-* [ENHANCEMENT] Added XFS inode operations to XFS metrics
-* [ENHANCEMENT] Remove CGO dependencies for OpenBSD amd64
-* [ENHANCEMENT] Added an ability to pass collector.supervisord.url via SUPERVISORD_URL environment variable
-* [BUGFIX] Handle EPERM for syscall in timex collector
+* [ENHANCEMENT]
 * [BUGFIX]
 
+## 1.1.0 / 2021-02-05
+
+* [CHANGE] Improve filter flag names. #1743
+* [CHANGE] Add btrfs and powersupplyclass to list of exporters enabled by default #1897
+* [FEATURE] Add fibre channel collector #1786
+* [FEATURE] Expose cpu bugs and flags as info metrics. #1788
```

Comment:
Can you group the collectors together?

Reply:
I'm not sure what you mean. They are order by PR #.

Reply:
Ok :) It was strange to see that in the middle of the new collectors.